src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -178,6 +180,47 @@ export const TravelDetailsMapScreenComponent = ({
   );
 };
 
+export type DirectionArrowParams = {
```

Comment:
I believe the common convention is to call it Props, not Params, i.e. `DirectionArrowProps` (or possibly just `Props`, as it's only used locally in this one component).